extensions/wikia/AdEngine/js/provider/btfBlocker.js

```diff
@@ -69,32 +64,54 @@ define('ext.wikia.adEngine.provider.btfBlocker', [
 			}
 		}
 
-		function startBtfQueue() {
-			log('startBtfQueue', log.levels.info.debug, logGroup);
+		function startSecondCallSlots() {
+			log('startSecondCallSlots', log.levels.info.debug, logGroup);
 
-			if (btfQueueStarted) {
+			if (secondCallQueueStarted) {
 				return;
 			}
 
-			lazyQueue.makeQueue(btfQueue, processBtfSlot);
-			btfQueue.start();
+			// highly viewable slots can be unblocked in second call
+			if (win.ads.runtime.unblockHighlyViewableSlots && config.highlyViewableSlots) {
+				config.highlyViewableSlots.forEach(function(slotName) {
+					log(['Unblocking HiVi slots', slotName], log.levels.info, logGroup);
+					unblock(slotName);
+				});
+			}
+
+			// ATF slots are always called in second call if not called in first
+			if (config.atfSlots) {
+				config.atfSlots
+				// remove slot already called in first call
+				.filter(function(slotName) {
+					return config.firstCallSlots.indexOf(slotName) === -1;
+				})
+				// unblock slot
```

Comment:
What about unblocking BTF slots?

Reply:
I really dislike the idea of variable `unblockedSlots`. Why not set the property `disabled` in slot config instead? What do you think about that?

Reply:
I deleted the previous comment even though it was right :)
So:
ATF slots are added to `unblockedSlots` to make sure they're always used.
BTF slots will be used if `isSecondCallEnabled()` returns value `true` https://github.com/Wikia/app/blob/ADEN-7702/extensions/wikia/AdEngine/js/provider/btfBlocker.js#L55

Reply:
Got it 👍